docs/ddl-of-carbondata.md

```diff
@@ -681,24 +682,24 @@ Users can specify which columns to include and exclude for local dictionary gene
 
      **NOTE:** Drop Complex child column is not supported.
 
-   - ##### CHANGE DATA TYPE
+   - ##### CHANGE COLUMN NAME/TYPE
    
-     This command is used to change the data type from INT to BIGINT or decimal precision from lower to higher.
+     This command is used to change the column's name and the data type from INT to BIGINT or decimal precision from lower to higher and rename column.
      Change of decimal data type from lower precision to higher precision will only be supported for cases where there is no data loss.
 
      ```
-     ALTER TABLE [db_name.]table_name CHANGE col_name col_name changed_column_type
+     ALTER TABLE [db_name.]table_name CHANGE old_col_name new_col_name column_data_type
```

Comment:
Change this as below
`ALTER TABLE [db_name.]table_name CHANGE col_old_name col_new_name column_type`